src/css/ng-wig.css

```diff
@@ -77,30 +77,25 @@
  *
  */
 .nw-editor {
-  display: block;
-  position: relative;
-  overflow-y: auto;
-  padding: 0 8px;
+  display: table;
```

Comment:
this is a fix for auto-expand feature that we've suddenly lost in one of the previous releases